pom.xml

```diff
@@ -120,6 +120,7 @@
         <module>presto-thrift-testing-server</module>
         <module>presto-thrift-connector</module>
         <module>presto-matching</module>
+        <module>presto-connector-tests</module>
```

Comment:
changes to root `pom.xml` should be part of `JUnit-5 unsupported test filtering by feature`. Right?